src/libsync/capabilities.cpp

```diff
@@ -176,6 +176,25 @@ bool Capabilities::chunkingNg() const
     return _capabilities["dav"].toMap()["chunking"].toByteArray() >= "1.0";
 }
 
+PushNotificationTypes Capabilities::pushNotificationsAvailable() const
```

Comment:
@er-vin @felixboehm 
I don't know why, but, for me, when I see this PushNotificationType**s** it makes me think of **PushNotificationTypes** as some sort of aggregate, while in fact, it's just an enum. I've immediately started to look for **push_back()** or **insert()**.

Reply:
Its's some sort of aggregate I think. It can hold multiple flags. How would you name it?

Reply:
@FlexW Got no idea right now :(

Reply:
Then I will leave it as it is?

Reply:
To be clear: I'm personally fine with it.

Now I can understand if it's unclear for someone else, which is what @allexzander expressed hence why I made proposals for alternative names.

@allexzander should @FlexW go for one of the proposals I made? Or we stay with "Types"?